benches/time_timeout.rs

```diff
@@ -0,0 +1,114 @@
+//! Benchmark spawning a task onto the basic and threaded Tokio executors.
+//! This essentially measure the time to enqueue a task in the local and remote
+//! case.
```

Comment:
Doesn't seem to be the text describing this benchmark? I guess it could be revised or removed.

Reply:
It is copied from the benchmark in `time.rs`. I think deleting this comment is acceptable for this benchmark.